lab_1/main.py

```diff
@@ -105,7 +195,14 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    context_list = []
+    for word_list in content:
+        s = ' '.join(word_list)
+        context_list.append(s)
+    context_str = '\n'.join(context_list)
+    name_file = path_to_file + 'report.txt'
```

Comment:
Path to file is passed as the following:
`path/to/file.txt` -> no need for this line.